packages/faucet/scripts/cli.ts

```diff
@@ -203,13 +200,10 @@ function setConfig(network: string, config: Partial<NetworkConfig & TwilioParams
     setIfPresent('node_url', config.nodeUrl),
     setIfPresent('faucet_gold_amount', config.faucetGoldAmount),
     setIfPresent('faucet_dollar_amount', config.faucetDollarAmount),
-    setIfPresent('gold_token_address', config.goldTokenAddress),
-    setIfPresent('stable_token_address', config.stableTokenAddress),
-    setIfPresent('escrow_address', config.escrowAddress),
     setIfPresent('invite_gold_amount', config.inviteGoldAmount),
     setIfPresent('invite_dollar_amount', config.inviteDollarAmount),
     setIfPresent('escrow_dollar_amount', config.escrowDollarAmount),
-    setIfPresent('expirary_seconds', config.expirarySeconds),
+    setIfPresent('expirary_seconds', config.expirySeconds),
```

Comment:
should we fix the string too? `'expirary_seconds'`

Reply:
Ah yep good catch, will fix